source/js/components/takeover/takeover.jsx

```diff
@@ -8,6 +9,12 @@ export default class Takeover extends React.Component {
       isHidden: true
     });
 
+    ReactGA.event({
+      category:`splash`,
+      action: `splash exit`,
+      label: `dismiss splash `
```

Comment:
For label, can distinguish between the two buttons? Using button string or add a name to element?